diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index 81a4c05..633cb9c 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -111,7 +111,6 @@ dependencies {
     implementation(platform(libs.androidx.compose.bom))
     implementation(libs.androidx.compose.ui)
     implementation(libs.androidx.compose.ui.graphics)
-    implementation(libs.androidx.graphics.path)
     implementation(libs.androidx.compose.ui.tooling.preview)
     implementation(libs.androidx.compose.material3)
     implementation(libs.androidx.compose.material3.windowsizeclass)
